DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md

```diff
@@ -1,7 +1,15 @@
 # IPFS metrics and usage statistics
-Open problems around metrics and usage statistics in IPFS can be grouped as follows:
+
+## Problem space
+1. Certain user groups (such as scientific research organizations, and binary/library package managers) will not adopt IPFS unless they can track metrics and usage statistics in as good or better ways as they do today. Today, download count is used as a proxy for usage. How could IPFS provide even better metrics/stats?
+1. How are applications using IPFS features today? How are those applications themselves being used?
+
+## Goal
+By the end of the session, we'd like to have a prioritized list of features that would most help applications built on top of IPFS, and provide that list to the IPFS development team to help focus their time and energy on highest-value work.
```

Comment:
...and an idea or proposal for how they might be implemented?

Reply:
hmm I agree with this. I don't want to come out of this deep dive with a list of nice to haves that aren't even remotely possible in a technical sense.

Reply:
I accepted @alanshaw 's suggestion -- thank you!